source/fe/mapping_fe_field.cc

```diff
@@ -58,13 +59,83 @@ MappingFEField<dim,spacedim,VECTOR,DH>::InternalData::InternalData (const Finite
 
 template<int dim, int spacedim, class VECTOR, class DH>
 std::size_t
-MappingFEField<dim,spacedim,VECTOR,DH>::InternalData::memory_consumption () const
+MappingFEField<dim,spacedim,VECTOR,DH>::InternalData::
+memory_consumption () const
 {
   Assert (false, ExcNotImplemented());
   return 0;
 }
 
 
+
+template<int dim, int spacedim, class DH, class VECTOR>
```

Comment:
You moved a bunch of functions to the .cc. Are any of these performance critical so inlining could matter?